spec/php-spec.coffee

```diff
@@ -1899,21 +1899,6 @@ describe 'PHP grammar', ->
     expect(lines[2][0]).toEqual value: 'HEREDOC', scopes: ['source.php', 'string.unquoted.heredoc.php', 'keyword.operator.heredoc.php']
     expect(lines[3][0]).toEqual value: ';', scopes: ['source.php', 'punctuation.terminator.expression.php']
 
-  it 'does not match incorrect heredoc terminators', ->
```

Comment:
Instead of removing these tests, they can be repurposed as valid examples of a heredoc.

Reply:
Thank you I'll try to run tests in order to make what you said earlier work properly.